src/main.rs

```diff
@@ -141,6 +158,12 @@ fn main() {
             if let Err(handled_pr) = handle_pull_request(pr, &bitbucket, &config.teamcity, &fanout) {
                 println!("{}{}", prefix(2), handled_pr);
             }
+
+            match &schedule {
+                &Some(ref sch) => sleep_duration = (sch.next_utc().unwrap() - UTC::now()).to_std().unwrap(),
+                _ => continue
+            }
```

Comment:
Hm, on second look, seems like yours may be a good idea... I have to think a little more...